packages/core/types/index.js

```diff
@@ -1329,7 +1330,10 @@ export interface BundleGraph<TBundle: Bundle> {
   /** Returns a list of bundles in the bundle graph. By default, inline bundles are excluded. */
   getBundles(opts?: {|includeInline: boolean|}): Array<TBundle>;
   /** Traverses the assets and dependencies in the bundle graph, in depth first order. */
-  traverse<TContext>(GraphVisitor<BundleGraphTraversable, TContext>): ?TContext;
+  traverse<TContext>(
+    GraphVisitor<BundleGraphTraversable, TContext>,
+    ?Asset,
+  ): ?TContext;
```

Comment:
There should be parameter nameslike `visitor: ..., startAsset: ?Asset`, similar to `traverseBundles`